packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,46 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-none), var(--surface-shadow-none);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  font-size: 14px;
+  color: var(--support-border-color-info);
+  padding: var(--layout-padding-large);
+  margin: var(--layout-margin-medium);
+  gap: var(--layout-margin-medium);
+  border: var(--layout-border-width-none);
+  text-align: left;
+  transition: box-shadow var(--transition-medium);
+}
+
+.ui-WalkthroughCard--Title {
+  color: var(--support-border-color-info);
+  font: var(--typography-bold-h5);
```

Comment:
This looks like it needs to be h4 for the homepage cards and h5 for the result page cards based on the design.